www/src/components/StickyResponsiveSidebar/StickyResponsiveSidebar.js

```diff
@@ -13,27 +13,43 @@
 
 import Container from 'components/Container';
 import {Component, React} from 'react';
+import PropTypes from 'prop-types';
 import Sidebar from 'templates/components/Sidebar';
 import {colors, media} from 'theme';
 import ChevronSvg from 'templates/components/ChevronSvg';
 
 class StickyResponsiveSidebar extends Component {
   constructor(props, context) {
     super(props, context);
-
-    this.state = {
-      open: false,
-    };
     this.toggleOpen = this._toggleOpen.bind(this);
   }
 
   _toggleOpen() {
-    this.setState({open: !this.state.open});
+    this.props.onRequestToggle(!this.props.open);
+  }
+
+  handleMenuTouchMove(evt) {
+    // Prevent evt.preventDefault in <MarkdownPage />
+    evt.stopPropagation();
```

Comment:
Again - this makes me a bit nervous, so I might take some time to think about possible problems we might run into by catching this event here.